README.md

```diff
@@ -13,6 +13,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.|
 | [ODSC (One Data Science Conference) ](https://india.odsc.com/) | 30 Aug – 2 Sep | Hotel Lalit Ashok, Bengaluru  | Conference on AI and Data Science |
+| [ReactFoo-Delhi 2018](https://reactfoo.in/2018-delhi/) | 18 Aug | Venue-TBA | ReactFoo Delhi Conference On React, alternatives to React, React Native and front-end engineering. |
```

Comment:
Some changes:

- In the first column, just `ReactFoo Delhi`. This list is for 2018, so we can skip the year (we should also be removing the year from other conferences in the list -- but that's another issue).
- For the venue, just `TBA`. The title of the column says "Venue" already. :)
- For the last column, `React , alternatives to React, React Native and front-end engineering.`.

Reply:
Sure @nikhita .. Will update the changes